src/pages/summary/summary.html

```diff
@@ -9,22 +9,23 @@
     <title>Kubernetes summary UI Web App</title>
     <link rel="stylesheet" href="./summary.css" type="text/css">
     <style>
-        .logout {
+        .k8s-ui-logout {
             position: absolute;
             right: 50px;
             top: 25px;
             float: right;
+            z-index: 3;
```

Comment:
this is required, in pod overview we have a header occupy the entire area, so making sure logout button is visible.